### ISSUE TITLE
Parallel TLC (workers > 1) reports intermittently an incorrect, too low  diameter

### ISSUE THREAD

**Issue by lemmy** [tlaplus/tlaplus#45]
Parallel TLC (workers > 1) reports intermittently an incorrect, too low  diameter (length of the longest behavior found so far).

Commit f72e862d74fa4f53fd589323babb2ece2872e170 contains a test case that intermittently on Linux x86_64 reproduces the bug.

Also, the branch [mku-trace-contention](https://github.com/lemmy/tlaplus/commits/mku-trace-contention) might have unintentionally fixed this. It changes the way how TLC handles the diameter internally.

**Comment by lemmy** [comment]
Indeed, mku-trace-contention never reports an incorrect, too low diameter.

**Comment by ahelwer** [comment]
Is this a duplicate of #883?
